client/extensions/woocommerce/style.scss

```diff
@@ -1,10 +1,64 @@
+.is-section-woocommerce {
+	.layout__primary {
+		display: flex;
+		justify-content: center;
+	}
+}
+
 .woocommerce {
+	flex-grow: 1;
+	max-width: 720px;
```

Comment:
@psealock @justinshreve Do you know how the stats page over wrote the max-width? Or why this page is having the full width layout applied to it?